Makefile

```diff
@@ -178,7 +177,13 @@ non-vulnerable-img:
 		-t ${NON_VULNERABLE_IMG} \
 		--target non-vulnerable .
 
-e2e-test: vulnerable-img eol-img non-vulnerable-img busybox-img collector-dummy-img
+custom-node:
+	docker build -t custom-node:latest \
+		-f test/e2e/test-data/Dockerfile.customNode \
+		--build-arg NODE_VERSION=${KUBERNETES_VERSION} test/e2e/test-data
+MODIFIED_NODE_IMAGE=custom-node:latest
```

Comment:
```suggestion
custom-node-$(KUBERNETES_VERSION):
	docker build -t custom-node:$(KUBERNETES_VERSION) \
		-f test/e2e/test-data/Dockerfile.customNode \
		--build-arg NODE_VERSION=${KUBERNETES_VERSION} test/e2e/test-data
MODIFIED_NODE_IMAGE=custom-node:$(KUBERNETES_VERSION)
```

Reply:
The reason for this is that we want to be sure the different images for the different kubernetes versions are distinct from each other and not overwriting each other.